packages/flutter/lib/src/material/refresh_indicator.dart

```diff
@@ -136,11 +137,26 @@ class RefreshIndicator extends StatefulWidget {
   /// Typically a [ListView] or [CustomScrollView].
   final Widget child;
 
-  /// The distance from the child's top or bottom edge to where the refresh
-  /// indicator will settle. During the drag that exposes the refresh indicator,
-  /// its actual displacement may significantly exceed this value.
+  /// The distance from the child's top or bottom [edge offset][edgeOffset] where
+  /// the refresh indicator will settle. During the drag that exposes the refresh
+  /// indicator, its actual displacement may significantly exceed this value.
+  ///
+  /// In most cases, [displacement] distance starts counting from the parent's
+  /// edges. However, if [edgeOffset] is larger than zero then the [displacement]
+  /// value is calculated from that offset instead of the parent's edge.
   final double displacement;
 
+  /// The offset where [RefreshProgressIndicator] starts to appear on drag start.
+  ///
+  /// Depending whether the indicator is showing on the top or bottom, the value
+  /// of this variable controls how far from the parent's edge the progress
+  /// indicator starts to appear. This may come in handy when, for example, the
+  /// UI contains a top [Widget] which covers the parent's edge where the progress
+  /// indicator would otherwise appear.
+  ///
+  /// By default, the edge offset is set to 0.
+  final double edgeOffset;
```

Comment:
It would be good to add a reference to the displacement property as well here. Something like:

```
/// See also: 
///
///   * [displacement], can be used to change the distance from the edge that the indicator settles.
```

Reply:
Done.